### PR TITLE
Link loading components

### PR DESCRIPTION
**Pull request by AaronPatterson1**

### CODE REVIEW

File: packages/_site/src/pages/components/loading/index.tsx [hunk 1]
```diff
@@ -98,6 +98,56 @@ export default function LoadingExample() {
         <LoadingSpinner size={LoadingSize.SMALL} />
 	);
 }
+`}
+                                />
+                            </div>
+                        </div>
+                        <div>
+                            <ExampleBlock
+                                title="Default"
```
Comment (clearfeld) [inline packages/_site/src/pages/components/loading/index.tsx]: If you're adding properties to the component than it isn't default, name it after the variant form.

File: packages/_site/src/pages/components/loading/index.tsx [hunk 2]
```diff
@@ -98,6 +98,56 @@ export default function LoadingExample() {
...
+        <LoadingSpinner size={LoadingSize.MEDIUM} />
+	);
+}
+`}
+                                />
+                            </div>
+                        </div>
+                        <div>
+                            <ExampleBlock
+                                title="Default"
```
Comment (clearfeld) [inline packages/_site/src/pages/components/loading/index.tsx:132]: If you're adding properties to the component than it isn't default, name it after the variant form.

File: packages/_site/src/pages/components/loading/index.tsx [hunk 3]
```diff
@@ -80,7 +80,7 @@ export default function LoadingPage() {
                         <div>
                             <ExampleBlock
                                 title="Default"
-                                description="The default form of a loading component."
+                                description="The small version of the loading component."
```
Comment (clearfeld) [inline packages/_site/src/pages/components/loading/index.tsx]: If this one is default leave the default in the description.

File: packages/_site/src/pages/components/link/index.tsx [hunk 4]
```diff
@@ -102,6 +102,33 @@ export default function LinkExample() {
...
+                                    code={`import React from 'react';
+
+import { Link } from "@controlkit/link";
+
+export default function LinkExample() {
+	return (
+		<Link href="/components/link/examples">Default</Link>
+	);
+}
+`}
```
Comment (clearfeld) [inline packages/_site/src/pages/components/link/index.tsx]: This won't show a visited Link by default. When a user views this a first time its going to look like a normal link until they actually visit it. This needs to have an explicit forced style and state to display correctly consistently.

**Review by clearfeld — CHANGES_REQUESTED** [review]

**Comment by AaronPatterson1** [comment]
Made requested changes. please check again.

File: packages/_site/src/pages/components/link/index.tsx [hunk 5]
```diff
@@ -84,7 +84,7 @@ export default function LinkPage() {
                             />
                             <div {...stylex.props(styles.codeWrapper)}>
                                 <PreviewBlock>
-                                    <Link href="https://github.com/clearfeld">Default</Link>
+                                    <Link href="https://github.com/clearfeld" style={{color: "#006699"}}>Default</Link>
```
Comment (clearfeld) [inline packages/_site/src/pages/components/link/index.tsx]: Use the css variable colors.

File: packages/_site/src/pages/components/link/index.tsx [hunk 6]
```diff
@@ -110,7 +110,7 @@ export default function LinkExample() {
                             />
                             <div {...stylex.props(styles.codeWrapper)}>
                                 <PreviewBlock>
-                                    <Link href="/components/link/examples">Visited</Link>
+                                    <Link href="/components/link/examples" style={{color: "#b6aef2"}}>Visited</Link>
```
Comment (clearfeld) [inline packages/_site/src/pages/components/link/index.tsx]: Same as above

**Review by clearfeld — CHANGES_REQUESTED** [review]

**Comment by AaronPatterson1** [comment]
Made changes, please check again.

**Review by clearfeld — APPROVED** [review]

**Pull request merged**